### PR TITLE
Add trigger to package nightly builds

### PR DESCRIPTION
**Pull request by joecummings**
See title

### CODE REVIEW

File: .github/workflows/build-wheels-linux.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux.yml@main
+    with:
+      # TODO #4: Enter the name of your repo
+      repository: pytorch/torchtune
+      ref: ""
+      # TODO #5: Have custom build steps before building the wheel? Pass the path to the shell script with those steps below.
+      pre-script: ""
+      # TODO #6: Same as above, but post-build steps. If you have none, just use "".
+      post-script: ""
+      # TODO #7: Want to verify the correctness of your binaries before distributing them? Add custom python smoke tests below.
```
Comment (NicolasHug) [inline .github/workflows/build-wheels-linux.yml:36]: Should there be small smoke tests here before merging, or do you prefer merging the PR first?

Comment (joecummings) [reply .github/workflows/build-wheels-linux.yml]: I added a small smoke test

File: .github/workflows/build-wheels-linux.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  generate-matrix:
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    with:
+      package-type: wheel
+      os: linux
+      test-infra-repository: pytorch/test-infra
+      test-infra-ref: main
+      with-cuda: disable
+      with-rocm: disable
+      with-cpu: enable
```
Comment (atalman) [inline .github/workflows/build-wheels-linux.yml:24]: no need to include with-cpu here, its enabled by default"
https://github.com/pytorch/test-infra/blob/main/.github/workflows/generate_binary_build_matrix.yml#L36

File: .github/workflows/build-wheels-linux.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  build:
+    needs: generate-matrix
+    name: ${{ matrix.repository }}
```
Comment (atalman) [inline .github/workflows/build-wheels-linux.yml:27]: Add also 
```
strategy:
      fail-fast: false
```
This way when 1 build failed, everything else can still continue

**Comment by joecummings** [comment]
Closing in favor of #646

**Pull request closed without merging**